src/components/modals/SharingModal.js

```diff
@@ -7,6 +7,10 @@ import { Modal, Button, OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { FaShareAlt } from 'react-icons/fa';
 import { FaShareAltSquare } from 'react-icons/fa';
 
+import { FaQrcode } from 'react-icons/fa';
+
+import QRCode from "react-qr-code";
```

Comment:
The animated lib didn't have its ES6 / import interface documented.

Reply:
Sounds good :+1: